docs/getting_started.md

```diff
@@ -216,6 +218,64 @@ Here are some resources that can be used to get you started working with `AnnDat
 - [Converting directly from R to Python](https://theislab.github.io/scanpy-in-R/#converting-from-r-to-python)
 - [Homepage for scanpy tutorials](https://scanpy.readthedocs.io/en/latest/tutorials.html)
 
+
+## Special considerations for CITE-seq experiments
+
+If the dataset that you have downloaded contains samples with ADT data from a CITE-seq experiment, the raw and normalized ADT expression matrices are stored in an `altExp` named `"adt"` of the `SingleCellExperiment` object in the `processed.rds` file:
+
+```r
+# View the ADT alternative experiment
+altExp(processed_sce)
+
+# You can also explicitly specify the altexp's name:
+altExp(processed_sce, "adt")
+```
+
+The following commands can be used to access the ADT expression matrices:
+
+```r
+# the raw ADT counts matrix
+raw_adt_counts <- counts(altExp(processed_sce))
+
+# the log normalized ADT counts matrix
+normalized_adt_counts <- logcounts(altExp(processed_sce))
+```
+
+Be aware that the `SingleCellExperiment` object in the `processed.rds` file was not filtered based on ADT counts.
+The `adt_scpca_filter` column indicates which cells should be removed before proceeding with downstream analyses of the ADT data, as determined by [`DropletUtils::CleanTagCounts()](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html).
+Specifically, this process identified cells with high levels of ambient contamination and/or high levels of negative control ADTs (if available).
+Cells are labeled either as `"Keep"` (cells to retain) or `"Remove"` (cells to filter out).
+
+To filter cells based on this column, use the following command:
+
+```r
+# Filter cells based on ADT QC statistics
+processed_sce <- processed_sce[, which(processed_sce$adt_scpca_filter == "Keep")]
+```
+
+The normalized ADT expression matrix only contains values for cells labeled as `"Keep"` in the `adt_scpca_filter` column.
+Any cells labeled `"Remove"` have `NA` values in the normalized expression matrix (see {ref}`processed adt data <processing_information:Processed ADT data>` for more details).
+
+
+### Filtering cells based on ADT quality control
+
+In both the `filtered.rds` and `processed.rds` files, quality-control statistics calculated by [`DropletUtils::CleanTagCounts()`](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html) are provided in the alternative experiment's `colData` and can be used for filtering.
+
+To perform the same filtering as we recommended above for the `processed.rds` file, you can use the following command:
+```r
+# Filter cells based on ADT QC statistics
+filtered_sce <- filtered_sce[, which(filtered_sce$adt_scpca_filter == "Keep")]
+```
```

Comment:
Just realizing that now these two statements seem a bit oddly ordered since one talks about `altExp` `colData` but then we don't use that information. I might swap the order, so we are talking about the `colData` in the context of custom filtering.

Reply:
Yeah, that statement made more sense placed there when this code used the `discard` column! I will rearrange text a bit.